lib/wasi/src/os/console/txt/about.md

```diff
@@ -0,0 +1,9 @@
+# Wasmer Terminal
+
+This terminal is an Wasmer powered terminal hosted in a browser which implements
+a basic operating system and is natively integrated with ATE and WAPM.
+
+For more information try:
+
+about wasmer
+about deploy
```

Comment:
We have "deploy" mentionned here, is that ok?

Reply:
leftovers - it can be removed

Reply:
Removed.